Apparently "ulong" expands to "unsigned long", which doesn't work
as a functional cast.  Causes a compile error in kdebindings/php-qt.

--- origsrc/qt-x11-opensource-src-4.5.3/src/corelib/tools/qhash.h	2009-09-29 06:01:37.000000000 -0500
+++ src/qt-x11-opensource-src-4.5.3/src/corelib/tools/qhash.h	2009-12-29 02:51:15.337619500 -0600
@@ -75,7 +75,7 @@ inline uint qHash(ulong key)
         return uint(key);
     }
 }
-inline uint qHash(long key) { return qHash(ulong(key)); }
+inline uint qHash(long key) { return qHash((ulong)(key)); }
 inline uint qHash(quint64 key)
 {
     if (sizeof(quint64) > sizeof(uint)) {
